src/script.ts

```diff
@@ -113,6 +113,17 @@ function enableNightMode(): void {
     document.body.classList.add("nightmode");
 }
 
+function updateClock(): void {
+    const m = new Date();
+    const clockTextElm = document.getElementById("clock-text");
+    if (clockTextElm == null) {
+        console.log("clock text element not found");
+    } else {
+        clockTextElm.textContent = (m.getHours() < 10 ? ("0" + m.getHours().toString()) : m.getHours())
+             + ":" + (m.getMinutes() < 10 ? ("0" + m.getMinutes().toString()) : m.getMinutes());
```

Comment:
I think this could benefit from organization that makes it more clear what's going on. Something like (pseudocode):

```js
textContent = (
    m.getHours() < 10 ?  ...
    + ':'
    + m.getMinutes < 10 ? ...
)
```
Like where each part of the string is on its own line.

But also maybe we could use some TypeScript magic to get actual time formatting? That would be way better